syntax = "proto3";

message Product{
    string id_product = 1;
    string name = 1;
    string userName = 1;
}

message TransactionResponse{
    int32 status_code = 1;
}

service ProductService{
    rpc AddProduct(Product) returns (TransactionResponse) {}
    rpc DeleteProduct(Product) returns (TransactionResponse) {}
}